package.json

```diff
@@ -13,7 +13,7 @@
     "build": "tsc",
     "gen-docs": "typedoc --readme none --mode modules --excludeNotExported --excludePrivate --exclude **/*_test.ts --out ./docs/api .",
     "pretest": "npm run posttest; ln -s node_modules bower_components",
-    "test": "wct -l chrome",
+    "test": "npm run lint && wct -l chrome",
```

Comment:
I'd recommend running lint after tests. Most of the time your editor will tell you about lint errors as you work.

I would recommend building before testing though. Something like:

```
"test": "npm run build && wct -l chrom && npm run lint",
```

Reply:
done. fyi a lot of our projects have the previous test command